docs/services/gcp-services-guide.md

```diff
@@ -110,12 +87,11 @@ helm init --service-account=tiller
 * Install the GCP stack into Crossplane using the [GCP stack
   section](../install-crossplane.md#gcp-stack) of the install guide.
 
-We will be using `gcp-infra-dev` namespace to store our cloud-specific resource
-classes, cloud resources and providers. But you can use any namespace name for
-that matter.
+To keep your resource configuration organized, start by creating a new
+directory:
+
 ```bash
-kubectl create namespace gcp-infra-dev
-export INFRA_NAMESPACE=gcp-infra-dev
+mkdir wordpress && cd $_
 ```
 
 #### GCP Provider
```

Comment:
Can we defer to https://crossplane.io/docs/master/cloud-providers/gcp/gcp-provider.html rather than re-documenting this? I believe that's what the AWS services guide did.